Makefile

```diff
@@ -1,5 +1,5 @@
 CI_BUILD_NUMBER ?= $(USER)-snapshot
-VERSION ?= 4.11.$(CI_BUILD_NUMBER)
+VERSION ?= 4.12.$(CI_BUILD_NUMBER)
```

Comment:
One could argue this is a "breaking" change because it will cause long strings of text without spaces (e.g.: the screenshots attached to this PR description) to overflow the `gridList-item`.
I don't think it's "breaking" enough to bump a major version, but I'm open to it if anybody thinks this we should be bumping a major version.